documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -3213,3 +3218,45 @@ The following would be valid:
 ...
 ```
 Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rules](#automated-rules) | [ARM](#arm-violations): [Errors](#arm-errors) or [Warnings](#arm-warnings) | [SDK](#sdk-violations): [Errors](#sdk-errors) or [Warnings](#sdk-warnings)
+
+### <a name="r4031"></a>R4031 RPaasResourceProvisioningState
```

Comment:
ResourceProvisionStateRequired. 

"RPaaS" should be the applied scope, no need to present in the name.  As already mentioned in the "Category".  So that if in the future, the scope is extended, no need to change the rule name.

Reply:
How about keep the current naming, as there are 3 existing Rpaas Rules whose name are also following this pattern, And for rpaas rules it's more strict , should not change it's scope in the future.